src/main/java/racing/car/car/Car.java

```diff
@@ -1,8 +1,23 @@
 package racing.car.car;
 
 public class Car implements Movable {
-    private int position;
+
     private static final int MOVE_THRESHOLD = 4;
+    private final String name;
+    private final String OVER_MESSAGE_ERROR = "자동차 이름이 5자글자를 초과하였습니다.";
+    private int position;
+
+    public Car(String name) {
+        checkName(name);
+        this.name = name;
+    }
```

Comment:
부 생성자는 주 생성자를 호출해서 중복을 제거해 보시면 좋을 것 같아요
```suggestion
    public Car(String name) {
        this(name, 0);
    }
```